### PR TITLE
Add `chainId` to EIP 1559 txs

### PR DESCRIPTION
**Pull request by Uxio0**
- `chainId` is mandatory for eip1559 txs
- Rename `eip1159` -> `eip1559`

### CODE REVIEW

File: .github/workflows/python.yml [hunk 1]
```diff
@@ -51,7 +51,6 @@ jobs:
     - name: Test setup.py
       run: pip install -e .
     - name: Send results to coveralls
-      if: ${{ env.COVERALLS_REPO_TOKEN }}
```
Comment (fmrsabino) [inline .github/workflows/python.yml:54]: Is this change intentional?

**Review by fmrsabino — APPROVED** [review]

File: .github/workflows/python.yml (hunk 1 above)
Comment (Uxio0) [reply .github/workflows/python.yml:54]: Yes, coveralls where not running
